deploy/kubernetes/setup-sumologic.yaml.tmpl

```diff
@@ -79,51 +83,56 @@ data:
+        name         = local.apiserver_metrics_source
         collector_id = "${sumologic_collector.collector.id}"
     }
-  
-    resource "sumologic_http_source" "apiserver_metrics_source" {
-        name         = local.apiserver-metrics-source-name
+    
+    resource "sumologic_http_source" "control_plane_metrics_source" {
+        name         = local.control_plane_metrics_source
         collector_id = "${sumologic_collector.collector.id}"
     }
-  
-    resource "sumologic_http_source" "events_source" {
-        name         = local.events-source-name
-        category     = "${var.cluster_name}/${local.events-source-name}"
+    
+    resource "sumologic_http_source" "controller_metrics_source" {
+        name         = local.controller_metrics_source
         collector_id = "${sumologic_collector.collector.id}"
     }
-  
-    resource "sumologic_http_source" "kube_controller_manager_metrics_source" {
-        name         = local.kube-controller-manager-metrics-source-name
+    
+    resource "sumologic_http_source" "default_metrics_source" {
+        name         = local.default_metrics_source
         collector_id = "${sumologic_collector.collector.id}"
     }
-  
-    resource "sumologic_http_source" "kube_scheduler_metrics_source" {
-        name         = local.kube-scheduler-metrics-source-name
+    
+    resource "sumologic_http_source" "kubelet_metrics_source" {
+        name         = local.kubelet_metrics_source
         collector_id = "${sumologic_collector.collector.id}"
     }
-  
-    resource "sumologic_http_source" "kube_state_metrics_source" {
-        name         = local.kube-state-metrics-source-name
+    
+    resource "sumologic_http_source" "node_metrics_source" {
+        name         = local.node_metrics_source
         collector_id = "${sumologic_collector.collector.id}"
     }
-  
-    resource "sumologic_http_source" "kubelet_metrics_source" {
-        name         = local.kubelet-metrics-source-name
+    
+    resource "sumologic_http_source" "scheduler_metrics_source" {
+        name         = local.scheduler_metrics_source
         collector_id = "${sumologic_collector.collector.id}"
     }
-  
+    
+    resource "sumologic_http_source" "state_metrics_source" {
+        name         = local.state_metrics_source
+        collector_id = "${sumologic_collector.collector.id}"
+    }
+    
     resource "sumologic_http_source" "logs_source" {
-        name         = local.logs-source-name
+        name         = local.logs_source
         collector_id = "${sumologic_collector.collector.id}"
     }
-  
-    resource "sumologic_http_source" "node_exporter_metrics_source" {
-        name         = local.node-exporter-metrics-source-name
+    
+    resource "sumologic_http_source" "events_source" {
+        name         = local.events_source
         collector_id = "${sumologic_collector.collector.id}"
+        category     = "${var.cluster_name}/${local.events_source}"
```

Comment:
this seems to revert the current behavior of being able to change the sourceCategory for events?

https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/helm/sumologic/conf/setup/sumo-k8s.tf#L56

Reply:
This is actually consistent with our current behavior since this file is for non-helm users

https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/kubernetes/setup-sumologic.yaml.tmpl#L95